Module_Postman/activities/Act6_APIs_for_Front-End_Devs/2.md

```diff
@@ -0,0 +1,24 @@
+<!--title={Intro to APIs in our website}-->
+
+<!--badges={}-->
+
+<!--concepts={}-->
+
+For this activity, we will be using five different HTTP requests to NASA's API to construct a simple website that does the following things:
+
+<ol>
+  <li>Get the Picture of the Day from NASA</li>
+  <li>Get Data pertaining to objects nearing earth</li>
+  <li>Get MARS Rover Curiosity pictures</li>
+  <li>Get MARS Rover Spirit pictures</li>
+ 	<li>Get MARS Rover Opportunity pictures</li>
+</ol>
+
+ The Postman documentation for NASA's API can be found at [Postman](https://documenter.getpostman.com/view/35240/SVmtxerV?version=latest). Open the link, and then click **Run in Postman** on the top right of your screen to open NASA's API collection in Postman.
```

Comment:
I know it's mentioned at the end of the last section but since this can be a confusing idea it might be worth just reiterating upfront here that you can generate API documentation from a Postman collection and that's what we're using..